src/dropdowns/dropdown-item.ts

```diff
@@ -0,0 +1,57 @@
+import {Directive, Input, HostBinding, HostListener, ElementRef} from 'angular2/core';
+import {NglDropdown} from './dropdown';
+
+@Directive({
+  selector: '[nglDropdownItem]',
+  host: {
+    'tabindex': '0',
```

Comment:
You are correct, but it was by mistake that the directive was placed there. It has been amended.